### ISSUE TITLE
Transfluncer - klear.com but transparent

### ISSUE THREAD

**Issue by DevArifHossain** [DevArifHossain/saas-idea-validator#15]
Influence marketing can be a great user acquisition, branding channel for startups but it comes with its trade offs. That's why we are seeing many startups working on this soace to make it easier. We even have a few local players in BD like [Hypescout](hypescout.co)

## Problem
After trying out some of these tools, One thing I see they are lacking is **transparency**.
In most case their process is either you book a demo with them or you have to submit a few forms and based on that they suggest you some influencers.
Which may be alright for large companies, but as a bootstrapping or early stage startup where cost of acquisition is such a big priority, I am not confident with this model!

## Solution
It would be great if could see this influencers first, their audience category, pricing (how much the individual charge per post). Kind of like a marketplace where You can pick a few influencers and create & launch marketing campaigns by yourself.
And that is what **Transfluencer** aims to do.

## Pricing
We will take a commission from influencers based on their earnings.

**Comment by omranjamal** [comment]
This is a great model, I still wonder how important transparency is though.

**Comment by omranjamal** [comment]
Let's goooo! In that case there's a market for this. Essentially giving a good UX for a influencer post marketplace.

**Comment by ariflogs** [comment]
Yeah. I'm being active on twitter and thinking of starting a vlogging channel on youtube to share my products building journey!
If I can build a small community, then guess we can launch an MVP for this.
